src/index.js

```diff
@@ -27,7 +27,7 @@ class BlockService {
    * @param {Bitswap} bitswap
    * @returns {void}
    */
-  goOnline (bitswap) {
+  setExchange (bitswap) {
```

Comment:
The `@param` note above should probably change to be {Exchange} exchange rather than hardcoded bitswap.

On another note, we should probably add some lightweight checks that the expected functions exists on the object that gets passed in, to prevent scenarios when you set an exchange then call `put` and get a error about that the function is undefined, we can fail as early as when setting the exchange, with a more friendly error.